atom/common/native_mate_converters/net_converter.cc

```diff
@@ -68,11 +69,72 @@ v8::Local<v8::Value> Converter<scoped_refptr<net::X509Certificate>>::ToV8(
             val->GetIntermediateCertificates().front(),
             issuer_intermediates);
     dict.Set("issuerCert", issuer_cert);
+
+    std::vector<std::string> intermediates_encoded;
+    for (size_t i = 0; i < intermediates.size(); i++) {
+      auto native_cert = intermediates[i];
+      std::string encoded;
+      net::X509Certificate::GetPEMEncoded(native_cert, &encoded);
```

Comment:
I'm a little torn on that. On the one hand, I like being careful. On the other hand, we don't have any way of signaling that "yooooo we couldn't encode this one." And if we encode some nonsense, that has the funny upside of failing when we `FromV8` the certificate. On the other hand, it's hard to know exactly what shape of nonsense we'll get if this fails.

What do you think?